PowerShell/ScubaGear/Modules/CreateReport/CreateReport.psm1

```diff
@@ -182,12 +182,26 @@ function New-Report {
     # Handle AAD-specific reporting
     if ($BaselineName -eq "aad") {
         $ReportHTML = $ReportHTML.Replace("{AADWARNING}", $AADWarning)
-        $ReportHTML = $ReportHTML.Replace("{CAPTABLES}", "")
+
+        # Create an array of custom objects for each SKU
+        $LicenseInfoArray = $SettingsExport.license_information | ForEach-Object {
+            [pscustomobject]@{
+                "SKU Part Number" = $_.SkuPartNumber
+                "Used Licenses" = $_.ConsumedUnits
+                "Total Licenses" = $_.PrepaidUnits.Enabled
+            }
+        }
+        # Convert the custom objects to an HTML table
+        $LicenseTable = $LicenseInfoArray | ConvertTo-Html -As Table -Fragment
```

Comment:
Mitchel has a change to the way we do this due to Section 508.  Can you seek feedback from him on the preferred direction?  Also not sure the timeline so this may be fine to go in as is.

Reply:
Sure, I will reach out to Mitchel

Reply:
Devesh and I discussed, the ConvertTo-Html cmdlet will be fine here.